firefoxThemes/popup/popup.css

```diff
@@ -201,3 +202,10 @@ label[for="hideSearch"] {
 .slider.round:before {
   border-radius: 50%;
 }
+
+#show-settings {
+  margin-bottom: 1rem;
+  color: var(--link-color);
+  cursor: pointer;
+  text-decoration: underline;
+}
```

Comment:
An `onhover` effect of some sort will provide affordance to `#show-settings`.

Reply:
I'll agree that it can enhance the UX a bit, but I am not a fan of the font size change & addition of word shadow. So I opted for a color change in place.

Reply:
Since the popup menu contains animated elements, it is only fair the `Show Settings` link also receives an animation on hover.

Reply:
> So I opted for a color change in place.

For some themes, the color change is difficult to see.

https://user-images.githubusercontent.com/23222943/148705895-871d5bfd-9b92-4ad5-950f-4990870672ec.mp4

.

Reply:
That's fair, how about just changing opacity

https://user-images.githubusercontent.com/15972415/148706777-7483086c-6aed-476f-a445-eeb226cffe58.mp4

I think I may eventually need to think of a hyperlink color for the doki theme.